app/ethermint.go

```diff
@@ -58,6 +59,7 @@ var (
 		crisis.AppModuleBasic{},
 		slashing.AppModuleBasic{},
 		supply.AppModuleBasic{},
+		evm.AppModuleBasic{},
```

Comment:
The AppModuleBasic isn't finished as of now, so it will error out to include now (nil pointer dereference error)

Reply:
Commented out